src/stage0.txt

```diff
@@ -12,9 +12,9 @@
 # source tarball for a stable release you'll likely see `1.x.0` for rustc and
 # `0.x.0` for Cargo where they were released on `date`.
 
-date: 2018-01-02
-rustc: beta
-cargo: beta
+date: 2018-02-12
```

Comment:
Is the date correct?

Reply:
Oh... It might not be since we promoted to non-dev channel later. Let me check.